libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -70,23 +70,30 @@ WasmDialect::WasmDialect()
 	addFunction("i64.extend_i32_u", {i32}, {i64});
 
 	addFunction("i32.store", {i32, i32}, {}, false);
```

Comment:
Does this end up being `memory: write`? (it should)

Reply:
Yes. By default, they have `memory`, `storage` and `otherState` as write.

Reply:
Where do they have that by default? At line 259 below?
(f.sideEffects = _movable ? SideEffects{} : SideEffects::worst();)

Reply:
@leonardoalt Yes. The choice of word `default` was wrong. I meant all builtins that are not movable will get `SideEffect::Write`.